docs/getting-started/user-guides/dcrctl-basics.md

```diff
@@ -49,71 +77,63 @@ If the command was successful, you will not get a confirmation from `dcrctl`, bu
 
 NOTE: Because unlocking the wallet is required for many functions of `dcrwallet`, `dcrwallet` can be started with the `--promptpass` flag or setting `promptpass=true` in `dcrwallet.conf` (discussed [here](/advanced/storing-login-details.md#dcrwalletconf)).
 
+---
+
 ## Checking Your Balance
 
 To send the `getbalance` command to `dcrwallet` using `dcrctl`, enter the following in your shell:
 
 ```no-highlight
-dcrctl -u [wallet username] -P [wallet password] --wallet getbalance
+dcrctl --wallet getbalance
 ```
 
 This will return all of the balances for all of your accounts.
 
+---
+
 ## Getting a New Receiving Address
 
-To send the `getnewbalance` command to `dcrwallet` using `dcrctl`, enter the following in your shell:
+To send the `getnewaddress` command to `dcrwallet` using `dcrctl`, enter the following in your shell:
 
 ```no-highlight
-dcrctl -u [wallet username] -P [wallet password] --wallet getnewaddress
+dcrctl --wallet getnewaddress
 ```
 
 This will generate and return a new payment address. If you wish to minimize address reuse, use this command to get a new address for each transaction you wish to recieve.
```

Comment:
Could we change this to say 'To minimize address reuse, you should use this command ...'?